k8s/mongodb/mongo-ss.yaml

```diff
@@ -72,7 +72,7 @@ spec:
         resources:
           limits:
             cpu: 200m
-            memory: 768Mi
+            memory: 3.5G
```

Comment:
Exactly, it accepts both. This is the guide I referred to before pushing the change.